### DISCUSSION TITLE
Issue with replicating a netbox instance - relation already exists

### DISCUSSION THREAD

**Discussion by jozefws** [netbox-community/netbox#12883] [Help Wanted!]
Hi there - Trying to replicate a netbox instance from an old server.

I have pg_dumped it to an SQL file and have a fresh brand new instance of netbox installed on the new machine. Going through the replication steps it all works fine until it gets to migrating to circuit provider.

Then the migration errors out and spits out `django.db.utils.ProgrammingError: relation "circuits_provideraccount" already exists`.
e.g. "manage.py migrate --fake"
I have tried all the obvious solutions from stack overflow which don't work. 

The netbox version on the new machine is v3.5.3 and the older machine was on one from 3.4.7.

Any ideas? Thanks.

**Comment by candlerb** [comment]
On the old (source) box, try `sudo -u postgres psql netbox` then `\d circuits_provideraccount`. Does the table exist? If it does, then it would appear that you already did a partial/failed upgrade to 3.5.x on this box.

If it doesn't (and you should also check this table doesn't exist in the pg_dump file that you exported), then the problem is likely on the import side: you need to completely drop and recreate the (empty) netbox database, before restoring from the pg_dump file.  *Then* you can run `./upgrade.sh` which will migrate it up to 3.5.x.

There should be no need to run `manage.py migrate`.  The `./upgrade.sh` script will take care of it for you.

**Reply by jozefws** [reply]
Hi there! Thanks for your reply.

Following the steps there was indeed a failed upgrade to 3.5.x on the source but there is a past export without the curcuits table.
I then tried to import this older export and it was all okay up until the following error:
`django.db.utils.IntegrityError: duplicate key value violates unique constraint "django_migrations_pkey"`
This reappeared after clearing the database and trying twice.
Any ideas?

Cheers.
